_docs/student/communication/markdown.md

```diff
@@ -94,4 +94,19 @@ using markdown.
    3. Item 3
    ```
 
+* **Table**
+
+    To add a table, use three or more hyphens `(---)` to create each column’s header, and use pipes `(|)` to separate each column. For compatibility, you should also add a pipe on either end of the row.
+
+    ```
+    To add a table, use three or more hyphens (---) to create each column’s header, and use pipes (|) to separate each column. For compatibility, you should also add a pipe on either end of the row.
+
+    | Syntax      | Description |
+    | ----------- | ----------- |
+    | Header      | Title       |
+    | Paragraph   | Text        |
+
+    ```
```

Comment:
```suggestion
    ```
    The resulting table would look like this:
    | Syntax      | Description |
    | ----------- | ----------- |
    | Header      | Title       |
    | Paragraph   | Text        |
```